runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/LabeledMetrics.java

```diff
@@ -29,7 +30,7 @@
   /**
    * Create a metric that can be incremented and decremented, and is aggregated by taking the sum.
    */
-  public static Counter counter(MonitoringInfoMetricName metricName) {
+  public static Counter counter(MetricName metricName) {
```

Comment:
Let's keep it as MonitoringInfoMetricName, since this is meant to be an implementation only used by SDK and RunnerHarness authors. For creating system style metrics with URN+labels,